src/components/InstanceMonitoring/_instance-monitoring.scss

```diff
@@ -0,0 +1,51 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/typography';
+@import 'carbon-components/scss/globals/scss/vars';
+
+.instance-card {
+  background-color: #fff;
+  width: 1167px;
```

Comment:
Does it make sense to make this relative to the containing element, or can we do any other way to accomodate different real-screen estate...?